scripts/bootstrap.js

```diff
@@ -35,7 +35,7 @@ ls("packages/*").forEach(function (loc) {
 
 var completed = false;
 
-var bar = new ProgressBar(":packagename ╢:bar╟", {
+var bar = !/^win/.test(process.platform) && new ProgressBar(":packagename ╢:bar╟", {
```

Comment:
Better yet, the progress bar should just be fixed on Windows though. The one that npm itself produces works fine, so there's a way.